app/__tests__/home.test.ts

```diff
@@ -1,26 +1,7 @@
-import { UITestBuilder } from '@antiwork/shortest';
+import { test } from '@antiwork/shortest';
 
 interface loginButton {
   url: string;
 }
 
-define('Validate Dasboard is accessible by users', async () => {
-  afterAll(async () => {
-    console.log('Clearing DB after all tests');
-  });
-  beforeAll(async () => {
-    console.log('Clearing DB before all tests');
-  });
-  new UITestBuilder<loginButton>('/')
-    .test('Validate that users can access the dashboard')
-    
-    .given('baseUrl', { url: 'http://localhost:3000' })
-    .when('Clicking on view dashboard button')
-    .expect('Should be able redirect to /dashboard and see the dashboard')
```

Comment:
The AI can assume the test has passed by being able to see the dashboard via computer use, so this isn't necessary IMO.

Reply:
Yes we're removing them.